src/entity/posts/PostAnalyticsHistory.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  OneToOne,
+  PrimaryColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import type { Post } from './Post';
+
+@Entity()
+export class PostAnalyticsHistory {
+  @PrimaryColumn({ type: 'text' })
+  id: string;
```

Comment:
opted to do another table, to conserve space and also not create dependency between between two tables if we need to back import data